packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -611,6 +634,9 @@ template std::unique_ptr<MachineThread> ArbCore::getMachineUsingStateKeys(
     const MachineStateKeys& state_data,
     ValueCache& value_cache) const;
 
+constexpr uint256_t old_machine_cache_interval = 100'000;
+constexpr size_t old_machine_cache_max_size = 10;
```

Comment:
I'm concerned that a 1 million gas won't be enough of a reorg window. Maybe 1 million and 20 would be safer? There's a bit of a measurement question here. My concern is mainly about having to jump back to before the window and ending up loading a machine. I'm way less worried about doing some duplicate computation